src/logger/console-logger.service.ts

```diff
@@ -118,12 +123,22 @@ export class ConsoleLoggerService implements LoggerService {
     isTimeDiffEnabled?: boolean,
   ): void {
     let output;
-    if (isObject(message)) {
-      output = `${color('Object:')}\n${ConsoleLoggerService.sanitize(
-        JSON.stringify(message, null, 2),
-      )}\n`;
+
+    // if skipColor is set, we do not use colors and skip sanitizing
```

Comment:
Maybe I'm not fully understanding what this is supposed to do, but this seems to me more lika a `skipSanetization` instead of a `skipColor`.

Reply:
The (current) main use of `skipColor` is to disable the coloration of log messages by the ConsoleLogger, to allow the caller to provide its own color. For the latter, sanitization must also be disabled, as that would strip the color escape-codes.